lib/msal-common/src/response/AuthenticationResult.ts

```diff
@@ -12,6 +13,7 @@ export class AuthenticationResult {
     uniqueId: string;
     tenantId: string;
     scopes: Array<string>;
+    account: IAccount;
     idToken: string;
     idTokenClaims: StringDict;
```

Comment:
FYI We found in v1 that this is not the type we should be using for this. See #1766

Reply:
We can resolve this as a separate PR.